src/app/views/initial-setup/initial-setup-tips/initial-setup-tips.component.html

```diff
@@ -1,28 +1,29 @@
 <div class="initial-setup-tips">
   <app-close-icon></app-close-icon>
-    <div *ngIf="os==='iOS'">
-    <h5>Create Shortcut:</h5>
-    <h5>Step 1:</h5>
+  <h5>Great you made it!</h5>
+  <h6>Here is a tip for creating a shortcut:</h6>
+  <div>
+    <h6>Step 1:</h6>
```

Comment:
Would remove the ':' from the step titles

Reply:
done